client/packages/common/src/intl/locales/en/common.json

```diff
@@ -184,5 +188,7 @@
   "placeholder.search-by-name": "Search by name",
   "placeholder.search-by-name-or-code": "Search by name or code",
   "success": "Success!",
-  "success.sync-settings": "Sync settings updated successfully!"
+  "success.sync-settings": "Sync settings updated successfully!",
+  "success.data-saved": "Record saved",
```

Comment:
the use of 'Record' - that's more of a technical term. I would go with 'Saved successfully' which is fairly generic - or 'Patient saved' if this is what it is for

Reply:
Yup, went with "Saved successfully", since it's the generic message returned by the JSONForms hook.